x-pack/platform/plugins/shared/security/public/management/roles/edit_role/privileges/kibana/privilege_summary/privilege_summary_table.tsx

```diff
@@ -210,11 +235,20 @@ export const PrivilegeSummaryTable = (props: PrivilegeSummaryTableProps) => {
       </EuiFlexGroup>
     );
 
+    const categoryPrivileges = Object.keys(privileges).reduce((acc, key) => {
```

Comment:
__question:__ any reason this suggestion didn't work? https://github.com/elastic/kibana/pull/204402#discussion_r1899561904